WcaOnRails/app/models/user.rb

```diff
@@ -731,12 +730,6 @@ def can_edit_registration?(registration)
     can_manage_competition?(registration.competition) || (registration.user_id == self.id && editable_by_user)
   end
 
-  def can_delete_registration?(registration)
-    # A registration can only be deleted by a user after it has been accepted if the organizers allow
-    can_delete_by_user = (!registration.accepted? || registration.competition.registration_delete_after_acceptance_allowed?)
-    can_manage_competition?(registration.competition) || (registration.user_id == self.id && can_delete_by_user)
-  end
-
```

Comment:
Whoops, this is a new feature coded by Richard that you potentially missed during rebase...?
In any case, these lines must not be deleted!

Reply:
I totally messed this up, I've fixed it!